src/festim/exports/surface_temperature.py

```diff
@@ -0,0 +1,102 @@
+import csv
+from dolfinx import fem
+import ufl
+import festim as F
+
+class SurfaceTemperature:
```

Comment:
I would be explicit an name it `AverageSurfaceTemperature`

Reply:
This should inherit from `SurfaceQuantity`

```suggestion
class SurfaceTemperature(F.SurfaceQuantity):
```